domains/carlostirado.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "carlostirado23",
+    "email": "mariotiradotovar@outlook.com"
+  },
+  "record": {
+    "CNAME": "https://carlostirado.vercel.app/"
```

```suggestion
    "CNAME": "carlostirado.vercel.app"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).